libraries/AP_Math/polyfit.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+  polynomial fitting class, originally written by Siddharth Bharat Purohit
```

Comment:
Perhaps we should run the auto-formatter across this.  It's kind of messy.

Reply:
It also doesn't follow a bunch of our conventions like `uint8_t` for loops.

Reply:
already fixed